extendedSample/src/main/java/info/mqtt/android/extsample/internal/Connection.kt

```diff
@@ -196,7 +196,7 @@ class Connection private constructor(
     }
 
     companion object {
-        private const val FOREGROUND = true
+        private const val FOREGROUND = false
```

Comment:
Why was this variable changed?

Reply:
There is no need anymore. This is the goal of this chance